_posts/2017-04-26-Relay-Mutations.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: post_longform
+title: Relay Mutations
+date: 2017-04-26
+categories: [relay, graphql, JavaScript, guest]
+author: nik
+---
+
+> Hey there everyone, it took us two years to [make our](https://github.com/artsy/metaphysics/pull/583) GraphQL implementation support any mutations and we still don't have any Relay mutations yet.
```

Comment:
Is this still true? All our mutations thus far are designed to work with the Relay spec, so even that first one you linked to has been amended to the Relay style.